tests/util/test_platform.py

```diff
@@ -0,0 +1,28 @@
+import platform
+
+from keyring.util.platform_ import (
+    config_root,
+    data_root,
+    _config_root_Linux,
+    _config_root_Windows,
+    _data_root_Linux,
+    _data_root_Windows,
+)
+
+
+def test_platform_Linux():
+    # rely on the Github Actions workflow to run this on different platforms
+    if platform.system() != "Linux":
+        return
```

Comment:
I think it's better to skip test, not return (which will make it appear as successful when it has not run). You can use this decorator for example:
https://docs.pytest.org/en/latest/how-to/skipping.html#id1